### DISCUSSION TITLE
How to check if docker has latest containers?

### DISCUSSION THREAD

**Discussion by AAndersn** [GRIDAPPSD/gridappsd-forum#96] [Q&A] [answered]
I am trying to check if my installation has the latest platform containers on my AWS server.

My docker-compose file shows `image: gridappsd/viz${GRIDAPPSD_TAG}`, but when I run `./run.sh -t develop`, it doesn't pull any updated containers. 

I tried `docker ps`, but it doesn't give me any info when was the date modified of the container.

On windows / WSL2, I can use the docker GUI to check the date info from dockerhub, but on a remote machine, the GUI isn't available. 

@poorva1209 , @tonya1 , @craig8 , what command line tool can I use to check if I have the latest containers?

**Comment by tonya1** [comment, marked as answer]
docker-compose pull
